inc/admin/users/class-userbulk.php

```diff
@@ -238,10 +238,10 @@ public function getBulkMessageSubtitle( $status ) : string {
 				$subtitle = __( 'The following user(s) could not be added.', 'users' );
 				break;
 			case self::USER_STATUS_INVITED:
-				$subtitle = __( 'An invitation email has been sent to the user(s) below. A confirmation link must be clicked before their account is created.', 'users' );
+				$subtitle = __( 'User(s) successfully added to this book.', 'users' );
```

Comment:
The translations functions should always have the text domain as the 2nd param. For this plugin it's `pressbooks`

https://github.com/pressbooks/pressbooks/blob/dd283c3728ef124fa328ae1b0120e064630a8f5f/pressbooks.php#L11

Reply:
See: https://developer.wordpress.org/reference/functions/__/